docs/guides/writing-tests.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Writing tests
+category: Guides
+order: 9
+---
+
+> This section uses `@instructure/ui-test-utils` testing library, plseae see the installaion and configuration [here](/#ui-test-utils)
+
+## The anatomy of a test
+
+The following code example presents an annotated version of a typical test. The terms in the comments will be explained over the course of this section.
+
+```javascript
+import React from "react"
+import {
+  mount,
+  stub,
+  wait,
+  expect
+  findWithLabel,
+  findWithText
+} from "@instructure/ui-test-utils"
+
+import MyComponent from "../index"
+import { setupFixtures, cleanupFixtures } from "./fixtures"
+
+// test suite:
+describe("<MyComponent/>",
```

Comment:
Yes, I tried but I have to eliminate some parts like `setupSixtures`  and `cleanupFixtures` because I had no idea about them. And everything else works fine.